src/bp/sdk/botpress.d.ts

```diff
@@ -1360,12 +1361,70 @@ declare module 'botpress/sdk' {
     node: string
   }
 
+  export type FormDataField = any
+
   export interface FormData {
     id?: string
     contentType?: string
-    [key: string]: FormData[] | any
+    [key: string]: FormDataField
   }
 
+  interface FormOption {
+    value: string
+    label: string
+    related?: FormField
+  }
+
+  interface FormContextMenu {
+    type: string
+    label: string
+  }
+
+  export interface FormDynamicOptions {
```

Comment:
There is too many interfaces related to forms. Please move them to a namespace so we can group them more easily